### ISSUE TITLE
Deleting files of a certain age/ date range

### ISSUE THREAD

**Issue by PiManASM** [drewminns/slack#7]
Would it be possible to delete all files > X days old? Spare the recent stuff, and still delete in bulk.

**Comment by drewminns** [comment]
I will look into it!

**Comment by cschultz1272** [comment]
Did you ever find a solution to this?

**Comment by itwasmattgregg** [comment]
This would be awesome!

**Comment by KeithHHill** [comment]
Really looking for a tool to be able to mass delete old files...this is really close but there isn't any kind of age selector.

**Comment by romeroluisjavier** [comment]
This would be awesome!

**Comment by PepeLopez** [comment]
I second this feature!!! <3

**Comment by bbi-frank** [comment]
This would bring the application up from being mildly handy to *extremely* useful.   I want to be able to nuke only stuff older than a week.   Cheers!  ;-)

**Comment by TamDenholm** [comment]
I agree this would be an excellent feature. Repo is on my watch list now.

**Comment by olejnjak** [comment]
This is really missing feature which would be absolutely awesome!

**Comment by grimble** [comment]
Need it too.
Just add time filter section under "What kind of files?" with items "Older than [date]" and "Till this [day|week|month|year]" and you will be the best.
